app/backend/src/couchers/models.py

```diff
@@ -926,17 +925,22 @@ class Cluster(Base):
         primaryjoin="and_(Cluster.parent_node_id == Node.id, Cluster.is_official_cluster)",
         backref=backref("official_cluster", uselist=False),
         uselist=False,
+        viewonly=True,
```

Comment:
I don't 100% understand why viewonly=True is needed here/fixes the warning

Reply:
It's because it has a complicated `primaryjoin` (so it's not controlled by SA: SA isn't able to tell what to insert if you tell it to add something into this relationship)

Reply:
That makes sense, thanks!